geomstats/_backend/pytorch/linalg.py

```diff
@@ -59,6 +59,13 @@ def norm(x, ord=2, axis=None):
     return torch.norm(x, p=ord, dim=axis)
 
 
+def solve_sylvester(a, b, q):
+    solution = np.vectorize(
+        scipy.linalg.solve_sylvester,
+        signature='(m,m),(n,n),(m,n)->(m,n)')(a, b, q)
+    return torch.from_numpy(solution)
```

Comment:
Otherwise, looking at the code here:
https://github.com/scipy/scipy/blob/d286f8525c16b2cd4e179dea2c77b6b09622aff9/scipy/linalg/_solvers.py#L30-L105

Could give ideas on how to re-implement in pytorch.

Reply:
In both cases it calls Fortran code. This might be an issue to look at with the ADT